.github/workflows/repo-stale.yaml

```
@@ -0,0 +1,30 @@
+name: Issue Stale Check
```

```suggestion
name: Issue stale check
```
For consistency with the "Merge conflict labeler" workflow.